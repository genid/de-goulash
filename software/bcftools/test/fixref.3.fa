>1
ACGGGGACAGATATGTTAACT
>2
ATGAGTGAATCAAGCACTATT
>3
AAATTCAGATACAGAATCTTT
>4
GATGGACAGGATGACCTCTAG
>5
GGTTAAATGTCAAGGTGAGCT
>6
ATGCTAATAAAACATTAAAGT
>7
ATGAGAAAGTCTGAGAGTGCA
>8
CTACATGACTCTTTATGTTAC
