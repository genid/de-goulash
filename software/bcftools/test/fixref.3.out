##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=1>
##contig=<ID=2>
##contig=<ID=3>
##contig=<ID=4>
##contig=<ID=5>
##contig=<ID=6>
##contig=<ID=7>
##contig=<ID=8>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE
1	11	rs1535632	A	T	.	.	.	GT	1/1
2	11	rs363334	C	G	.	.	.	GT	0/0
3	11	rs7101540	A	T	.	.	.	GT	0/0
4	11	rs7113791	A	T	.	.	.	GT	1/1
5	11	rs778833	C	G	.	.	.	GT	1/1
6	11	rs4933195	A	T	.	.	.	GT	0/0
7	11	rs903997	C	G	.	.	.	GT	0/0
8	11	rs1942968	C	G	.	.	.	GT	1/1
